tests/util/test_logger.py

```diff
@@ -21,13 +22,36 @@ def _csv_to_dict(csv_path: str) -> dict:
     return result
 
 
+def _json_to_dict(json_path: str) -> dict:
```

Comment:
Took me a couple of minutes to figure out what this function does, consider adding a (brief) docstring?